src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -584,6 +605,30 @@ public function iShouldBeAbleToSelectPaymentMethod(string $paymentMethodName): v
         Assert::notFalse(array_search($paymentMethodName, array_column($paymentMethods, 'name'), true));
     }
 
+    /**
+     * @Then I should see :firstPaymentMethodName and :secondPaymentMethodName payment methods
+     */
+    public function iShouldSeeAndPaymentMethods(string $firstPaymentMethodName, string $secondPaymentMethodName): void
```

Comment:
I would name it `iShouldSeePaymentMethods` the plural form should be enough to give the hint of multiple ones. And the `And` here makes the function name hard to read

Reply:
it is an autogenerated name by behat be yea, in this case, I should change it :)